ch-06/src/Solution618.java

```diff
@@ -0,0 +1,45 @@
+import java.util.Scanner;
+
+public class Solution618 {
+    public static void main(String[] args) {
+        final Scanner input = new Scanner(System.in);
+        final String password = input.nextLine();
+
+        passwordChecker(password);
+    }
+    public static boolean isPasswordLongEnough(String str) {
+        return str.length() >= 10;
+    }
+
+    public static boolean isPasswordOnlyContainsLettersAndDigits(String str) {
+        for (int i = 0; i < str.length(); i++) {
+            boolean isCharAlphabetic = Character.isAlphabetic(str.charAt(i));
+            boolean isCharDigit = Character.isDigit(str.charAt(i));
+            if (!isCharAlphabetic && !isCharDigit) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    public static boolean isPasswordContainsThreeDigits(String str) {

```

Comment:
Little bit of abstraction:

In the current solution, the number of digits is hard-wired into the name of the method. This leaks the information that the method checks for three digit occurrences. To avoid leaking implementation details, it might be a better idea to name the method `isPasswordContainsEnoughDigits`. In that case, we can freely change the number of required digits in the method body, without breaking the contract between the method's name and its implementation.